src/impl_x86__base_implementation.inl

```diff
@@ -695,11 +699,12 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
         return AMD_PUMA;
       case CPUID(0x17, 0x01):
       case CPUID(0x17, 0x11):
-      case CPUID(0x17, 0x18):
       case CPUID(0x17, 0x20):
+      case CPUID(0x18, 0x00):
```

Comment:
This cpuid used only for Hygon processors and we already have in branch `IsVendorByX86Info(info, CPU_FEATURES_VENDOR_HYGON_GENUINE)`:
http://users.atw.hu/instlatx64/HygonGenuine/HygonGenuine0900F02_Hygon_CPUID.txt
http://users.atw.hu/instlatx64/HygonGenuine/HygonGenuine0900F11_Hygon_01_CPUID.txt
http://users.atw.hu/instlatx64/HygonGenuine/HygonGenuine0900F02_Hygon_CPUID3.txt
we define by vendor and this only applies to HygonGenuine